conf/messages.en

```diff
@@ -858,6 +858,12 @@ answerPage.section.assets.heading = Assets
 answerPage.section.moneyAsset.subheading = Money
 answerPage.section.shareAsset.subheading = Share
 answerPage.agentClientRef = Client reference number: {0}
+shareClass.title = What class are the shares
+shareClass.heading = What class are the shares
+shareClass.class = Ordinary
+shareClass.class = Preference
+shareClass.checkYourAnswersLabel = What class are the shares
+shareClass.error.required = Select shareClass
```

Comment:
Is this a bad merge?

Reply:
Removed these to follow the shareClass messages being used in master